benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -157,6 +157,12 @@ protected BenchmarkServiceFutureStub build(io.grpc.Channel channel,
     }
   }
 
+  public static abstract class AbstractBenchmarkService implements BenchmarkService, io.grpc.stub.BindableService {
```

Comment:
@carl-mastrangelo, I was thinking that the abstract class with default method implementations could be the _same_ base class as here. It'd be what we suggest everyone to use. I'd also expect a `ServerBuilder.addService(BindableService)` convenience overload.

Any trailing concerns I have are more nit-realm, like does "bind" still make sense (vs, say, getDescriptor) or should `bindService()` be able to return multiple descriptors. I didn't bother mentioning them, but if a tweak like that would make the change make more sense we could talk about it.